tests/e2e/Identity.test.ts

```diff
@@ -29,18 +33,49 @@ describe('e2e', () => {
       const relayWpUrl = utils.buildApiUrl(wsProtocol, host, port, path)
       relayProvider = new RelayProvider(relayApiUrl, relayWpUrl)
 
+      identityWallet = new IdentityWallet(relayProvider)
+
       DEFAULT_PASSWORD = 'ts'
       ACCOUNT_KEYS = ['address', 'serializedWallet', 'pubKey']
+      AMOUNT_KEYS = ['raw', 'value', 'decimals']
+    })
+
+    describe('Deploy identity', () => {
+      it('should deploy an identity contract when creating an identity account', async () => {
+        const createdAccount = await identityWallet.createAccount(
+          DEFAULT_PASSWORD
+        )
+        assert.hasAllKeys(createdAccount, ACCOUNT_KEYS)
+        expect(createdAccount.address.length).to.equal(42)
+        expect(createdAccount.address.slice(0, 2)).to.equal('0x')
+      })
     })
 
-    it('should deploy an identity contract when creating an identity account', async () => {
-      const identityWallet = new IdentityWallet(relayProvider)
-      const createdAccount = await identityWallet.createAccount(
-        DEFAULT_PASSWORD
-      )
-      assert.hasAllKeys(createdAccount, ACCOUNT_KEYS)
-      expect(createdAccount.address.length).to.equal(42)
-      expect(createdAccount.address.slice(0, 2)).to.equal('0x')
+    describe('Interact with identity', () => {
+      before(async () => {
+        await identityWallet.createAccount(DEFAULT_PASSWORD)
+      })
+
+      it('should relay meta transaction and return a transaction hash', async () => {
+        const rawTx: RawTxObject = {
+          data: '0x',
+          from: identityWallet.address,
+          nonce: 1,
+          to: identityWallet.address,
+          value: 0
+        }
+
+        const transactionHash = await identityWallet.confirm(rawTx)
```

Comment:
This is the transactionHash of the ethereum transaction? Is there a way to get the hash of the metaTransaction?